libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorDirectShip.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from 'react';
+import { i18n } from '../../../core';
+import { Availability } from '../../../product';
+import { DIRECT_SHIP_LABEL, DIRECT_SHIP } from '../../constants';
+import { itemRow, itemColumn } from './FulfillmentSelectorItem.style';
+import { useFulfillmentSelectorState } from './FulfillmentSelector.hooks';
+
+/**
+ * Renders the direct ship item label.
+ * @returns {JSX}
+ */
+export function FulfillmentSelectorDirectShip() {
+  const { productId } = useFulfillmentSelectorState();
+
+  return (
+    <div className={itemRow}>
+      <div className={itemColumn}>
+        {i18n.text(DIRECT_SHIP_LABEL)}
+      </div>
+      <div className={itemColumn}>
+        <Availability productId={productId} fulfillmentSelection={DIRECT_SHIP} />
+      </div>
+    </div>
```

Comment:
In case Direct Ship is not available (libraries/engage/locations/components/FulfillmentSelector/FulfillmentSelectorItemDirectShip.jsx), there was a logic to show appropriate message.
```
if (selected && !isOrderable) {
    return (
      <Availability
        className={radioItemLabel}
        showWhenAvailable
        text={i18n.text('product.fulfillment_selector.error.direct_ship_not_orderable')}
        state={AVAILABILITY_STATE_ALERT}
      />
    );
  }
```

Reply:
Done